src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -474,6 +477,19 @@ struct mlx5_cqe64 *uct_ib_mlx5_iface_cqe_unzip(uct_ib_mlx5_cq_t *cq)
     return title_cqe;
 }
 
+static void
+uct_ib_mlx5_update_cqe_compression_stats(uct_ib_iface_t *iface, uct_ib_mlx5_cq_t *cq) {
+#if ENABLE_STATS
```

Comment:
Minor: `#ifdef`.
Currently both `#if` and `#ifdef` are used throughout UCX. But let's at least not add more inconsistency (you used `#ifdef` above).